src/main/java/org/kohsuke/github/GHPullRequestReview.java

```diff
@@ -102,7 +103,7 @@ public GHPullRequestReviewState getState() {
 
     @Override
     public URL getHtmlUrl() {
-        return null;
+        return GitHubClient.parseURL(html_url);
```

Comment:
back compatibility saved. If field is not present, the return value is still `null`